initial commit packaging influxdb work

nyhu opened the pull request.

nyhu committed:
initial commit packaging influxdb work

antoinegelloz reviewed (changes requested):
Good start! 💯 👍  Maybe could you add some basic instructions on how to use the package?

Commit:
use original licence in main as it is the only peace of code related to influx work

Commit:
adding usage to readme

nyhu commented:
I added usage in readme as requested, I also added original licence in main.go in the file header as it's the only original file.
I created our own licence for the rest of the code, still under MIT.

antoinegelloz commented on README.md:
```diff
@@ -0,0 +1,83 @@
+A golang generative framework that palliate the lack of generic in go.
+
+It's for now a fork / package of https://github.com/influxdata/influxdb/tree/master/tools/tmpl 
+It remove support for .json comments wich is an unsafe / non ide proof way to derivate json spec.
```
```suggestion
It removes support for .json comments which is an unsafe / non IDE-proof way to derive from json specs.
```

antoinegelloz commented on README.md:
```diff
@@ -0,0 +1,83 @@
+
+Usage is inspired by test package
+
+After installing genious with go get or go install, use a go generate directive:
```
```suggestion
After installing genius with `go get` or `go install`, use a `go generate` directive:
```

nyhu committed:
Update README.md

Co-authored-by: Antoine Gelloz <antoine.gelloz@gmail.com>

nyhu committed:
Update README.md

Co-authored-by: Antoine Gelloz <antoine.gelloz@gmail.com>
